src/app/components/StakingDateSelector/index.tsx

```diff
@@ -193,8 +167,26 @@ export function StakingDateSelector(props: Props) {
           );
         })}
       </div>
-      <div className="sliderMonth tw-mt-5 tw-pr-0">
-        <Slider {...settingsSliderMonth}>
+      <div className="sliderMonth tw-mt-5 tw-pr-0 tw-relative">
```

Comment:
Please remove the global class sliderMonth, there is no need for it.

```suggestion
      <div className="tw-mt-5 tw-pr-0 tw-relative">
```

Reply:
Thanks for reminder, I also deleted some global styles that we were using with `react-slick`.